server/events/approve_policies_command_runner.go

```diff
@@ -91,9 +95,23 @@ func (a *ApprovePoliciesCommandRunner) buildApprovePolicyCommandResults(ctx *com
 	// Check if vcs user is in the owner list of the PolicySets. All projects
 	// share the same Owners list at this time so no reason to iterate over each
 	// project.
-	if len(prjCmds) > 0 && !prjCmds[0].PolicySets.IsOwner(ctx.User.Username) {
-		result.Error = fmt.Errorf("contact policy owners to approve failing policies")
-		return
+	if len(prjCmds) > 0 {
```

Comment:
Please add tests for this

Reply:
Thank you for your contribution.

It's unfortunate there weren't previous tests added because a change like this has the potential of breaking existing behavior. This is why it's always a good idea to include tests in your PR.

If you join the slack community, the #contributors channel could help guide you on the tests if you're unfamiliar.

Reply:
Please use the pr template to include the tests section.

Reply:
After that PR is merged, we'd also need a test that checks the additional logic coming from this PR.

Reply:
Yep, absolutely 👍